packages/cxl-ui/src/components/cxl-dashboard-header.js

```diff
@@ -0,0 +1,114 @@
+  static get styles() {
+    return [cxlDashboardHeaderStyles];
+  }
+
+  @property({ type: String }) name = '';
+
+  @property({ type: String }) notificationCount = '6';
+
+  @property({ type: String }) updateTitle = 'What\'s new in CXL';
+
+  @property({ type: String }) lastCourseTitle = 'Advanced experimentation analysis';
+
+  @property({ type: String }) allCourses = 'Browse all courses';
+
+  @property({ type: String }) fastSkillsCourses = 'Browse fast skills';
+
+  @property({ type: String }) createRoadmap = 'Create your personal learning roadmap';
+
+  @property({ type: Boolean }) hasRoadmap = false;
+
+  renderLastCourse() {
+    return this.hasRoadmap
+    ? html`
+      <div class="last-course">
+        <vaadin-button>
+          <div>
+            <span class="subtitle">Continue where you left off</span>
+            <span class="title">${this.lastCourseTitle}</span>
+          </div>
+          <vaadin-icon icon="lumo:arrow-right"></vaadin-icon>
+        </vaadin-button>
+        <div class="progress">
+          <span class="progress-title">Completed 2 lessons of 6 lessons</span>
+          <vaadin-progress-bar value="0.334">Completed 2 lessons of 6 lessons</vaadin-progress-bar>
+        </div>
+      </div>`
+    : nothing;
+  }
+
+  renderStats() {
+    return this.hasRoadmap
+    ? html`
+      <div class="stats">
+        <slot name="stats-desktop"></slot>
+        <vaadin-details theme="cxl-dashboard-header reverse" class="vaadin-details">
+          <div slot="summary">Your roadmap</div>
+          <slot name="stats-mobile"></slot>
+        </vaadin-details>
+      </div>`
+    : nothing;
+  }
+
+  render() {
+    return html`
+      <div class="container">
+        <header>
+          <vaadin-button class="updates">
+            <vaadin-icon class="icon-bell" count="${this.notificationCount}" icon="lumo:bell"></vaadin-icon>
+            ${this.updateTitle}
+            <vaadin-icon icon="lumo:angle-right"></vaadin-icon>
+          </vaadin-button>
+          <div>
+            <span class="subtitle">My dashboard</span>
+            <h1 class="title">Hello ${this.name}!</h1>
+          </div>
+        </header>
+        <section class="content">
+          <div>
+            ${this.renderLastCourse()}
+            <div class="courses">
+              <vaadin-button>
+                ${this.allCourses}
+                <vaadin-icon icon="lumo:arrow-right"></vaadin-icon>
+              </vaadin-button>
+              <vaadin-button>
+                ${this.fastSkillsCourses}
+                <vaadin-icon icon="lumo:arrow-right"></vaadin-icon>
+              </vaadin-button>
+              ${!this.hasRoadmap
+                ? html`
+                  <vaadin-button class="roadmap">
+                    ${this.createRoadmap}
+                    <vaadin-icon icon="lumo:arrow-right"></vaadin-icon>
+                  </vaadin-button>`
+                : nothing
+              }
```

Comment:
I can't find on Figma state for no roadmap but instead of showing button I'd display the same stats component just with `-` and button saying "Create roadmap" instead of "Edit"

Reply:
To me these are two completely different scenarios. One is when user has some ongoing roadmaps/courses and the second is when there isn't any. Could you explain your suggestion, I'm not sure if I got it right. 

<img width="706" alt="Screenshot 2023-07-31 at 16 31 01" src="https://github.com/conversionxl/aybolit/assets/119847199/059d780e-ce82-4bf5-80f1-3b18e7601b17">
<img width="693" alt="Screenshot 2023-07-31 at 16 31 08" src="https://github.com/conversionxl/aybolit/assets/119847199/7dcb4c3c-6c3a-4cad-a26a-6c0408920f0c">

Reply:
You're right @kertuilves. I didn't see then second design in Figma before